doc/whats-new.rst

```diff
@@ -40,7 +40,9 @@ Bug fixes
 
 Documentation
 ~~~~~~~~~~~~~
-
+- Fix `variables` arg typo in `Dataset.sortby()` docstring
+ (:issue:`8663`, :pull:`8670`)
```

Comment:
```suggestion
  (:issue:`8663`, :pull:`8670`)
```
Is this why the doc build is failing? (not confident...)

Reply:
Looks like the RTD build passed with that change, but now the Benchmark results failed this time. Maybe a re-run will sort it out?